recipe/vina.patch

```diff
@@ -56,34 +45,47 @@ index cf9d426..51609bd 100644
  
  LDFLAGS = -L$(BASE)/lib -L.
  
-@@ -21,35 +21,35 @@ LDFLAGS = -L$(BASE)/lib -L.
- # install libboost_thread-mt, but not
+@@ -22,9 +22,9 @@ LDFLAGS = -L$(BASE)/lib -L.
  # libboost_thread (e.g. macOS)
  $(shell echo "int main(){ return 0; }" > linktest.cpp)
+ ifeq ($(BOOST_STATIC), y)
+-$(shell $(CC) $(LDFLAGS) ${BASE}/lib/libboost_thread-mt${BOOST_LIB_VERSION}.a linktest.cpp -o linktest >/dev/null 2>&1)
++$(shell $(VINA_CC) $(LDFLAGS) ${BASE}/lib/libboost_thread-mt${BOOST_LIB_VERSION}.a linktest.cpp -o linktest >/dev/null 2>&1)
+ else
```

Comment:
I'd prefer not to even patch the `BOOST_STATIC=y` case. It should just fail, because we want to build against the shared libs (or just the headers, if that suffices).